config.json

```diff
@@ -251,7 +251,7 @@
         "name": "Pascals Triangle",
         "uuid": "99493160-4673-402f-acda-62db5378148d",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["arrays", "for-loops"],
```

Comment:
I don't know if we'll have for-loops as separate concept. Maybe just loops?

Reply:
Looks like we are heading towards having a `for-loops` concept => <https://exercism.lol/tracks/javascript/concepts/for-loops> :thinking:

Reply:
Then I can be ignored ;)